src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -103,7 +109,8 @@ public Person toModelType() throws IllegalValueException {
         final Address modelAddress = new Address(address);
 
         final Set<Tag> modelTags = new HashSet<>(personTags);
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelTags);
+        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelTags)
+            .withSocialMedia(socialMedia);
```

Comment:
May I know how this part is supposed to work? Right now seems like its not being read:
the Telegram field has a value of "jasonjason"
![image](https://user-images.githubusercontent.com/92777033/224904896-a3d9022f-c560-4dca-ac37-5bcf28c6f5b7.png)
but it's not shown here after I close and open the app:
![image](https://user-images.githubusercontent.com/92777033/224905099-08f0cc01-bdcb-411e-9d0b-2085644496fc.png)